### ISSUE TITLE
The Model is unstable

### ISSUE THREAD

**Issue by ysuzalmon** [carpedm20/DCGAN-tensorflow#179]
After training my own dataset, the model began to become unstable. The effect after visualization as shown:The output of d and d_ begins collapse after the 8 epochs.
Excuse me, what is the cause of this?

**Comment by ysuzalmon** [comment]
![screenshot from 2017-05-27 14 44 34](https://cloud.githubusercontent.com/assets/28341077/26485042/e5f6bade-4227-11e7-8ba9-8cd372374680.png)

**Comment by georgiazhang** [comment]
@ysuzalmon  hi ,can you please tell me how to draw the loss figure?

**Comment by ysuzalmon** [comment]
I use the Tensorboard to visualize.



1967314312@qq.com
 
From: georgiazhang
Date: 2017-05-28 19:55
To: carpedm20/DCGAN-tensorflow
CC: ysuzalmon; Mention
Subject: Re: [carpedm20/DCGAN-tensorflow] The Model is unstable (#179)
@ysuzalmon hi ,can you please tell me how to draw the loss figure?
